### DISCUSSION TITLE
Are the shims on these websites minshimes or crew

### DISCUSSION THREAD

**Discussion by pazza2234** [3kh0/ext-remover#460] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
are the shims on https://files.ultimatesrv.com/, or https://mirror.the-repo.org/RMA/Shims/ crew shims? I am looking for them, and cant find any because https://obelous.dev/ is not up. also if it matters I am looking for octopus shims.

**Comment by 3kh0** [comment, marked as answer]
They are crew, but there is not difference, minishim is the same thing, but a smaller file size

**Reply by amelia-bartlett** [reply]
A) the folks behind sh1mmer stopped distributing pre-patched shims due to legal concerns
B) these are stock, unmodified, official RMA shims, and thus neither.
C) crew and minishim are different - crew has more features such as an x11 desktop environment built into the shim.
D) you need to modify them yourself to get a sh1mmer shim, which version of chromebrew you use (at least if you use the [local](https://github.com/CoolElectronics/sh1mmer/tree/beautifulworld/wax) i.e. not web builder) determines whether they are crew or minishim

**Comment by amelia-bartlett** [comment]
They are stock RMA shims, you have to patch them yourself.

i.e. neither.
